cli/pkg/bind/plugin/bind.go

```diff
@@ -63,6 +71,7 @@ func (b *BindOptions) BindFlags(cmd *cobra.Command) {
 
 	cmd.Flags().StringVar(&b.APIBindingName, "name", b.APIBindingName, "Name of the APIBinding to create.")
 	cmd.Flags().DurationVar(&b.BindWaitTimeout, "timeout", time.Second*30, "Duration to wait for APIBinding to be created successfully.")
+	cmd.Flags().StringSliceVar(&b.AcceptedPermissionClaims, "permission-claims", nil, "List of accepted permission claims for the APIBinding. Format: --permissionClaims resource=secrets,all=true,state=Accepted")
```

Comment:
```suggestion
	cmd.Flags().StringSliceVar(&b.AcceptedPermissionClaims, "permission-claims", nil, "List of accepted permission claims for the APIBinding. Format: --permission-claims resource=secrets,all=true,state=Accepted")
```

Reply:
missing a way to specify a group. Also unclear how to decline.

Reply:
And how to accept multiple.

And how to accept all?

Reply:
The tests look very different. They use `:`.

Reply:
And remind me what `all=true` means.

Reply:
Is the command idem-potent to accept new claims on day 2?

Reply:
Alternative: `--accept-permission-claims foo.acme.com,bar.acme.com`

Reply:
> And remind me what all=true means.
accepted all resources. Kinda hack for non-working selectors :D we slowly working to fix this.

Reply:
> Is the command idempotent to accept new claims on day 2?
will create an issue for later. Kinda need this for demo now so need something in now :/ bind is not independent by current design.

Reply:
> accepted all resources. Kinda hack for non-working selectors :D we slowly working to fix this.

Wouldn't that be the default on the command line?

Reply:
yes, now yes